src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2775,6 +2777,71 @@ def load_arguments(self, _):
             action="store_true",
         )
 
+# pylint: disable=line-too-long
```

Comment:
May I ask why these lines of code need to be commented out? If the parameters should be removed, please delete them directly.

Reply:
We comment them out because these code will be introduced by one week or so later.
Currently we are under private preview to not expose this feature the user, unless the private preview looks good.

Reply:
Removing the agent subcommand from commands.py requires us to remove the parameters and helps to meet the requirement of style check.
Keeping them here so that we can bring them back soon. If you are against from commenting so many code, I am fine to remove them directly.

Reply:
Got it. Just commenting them out temporarily is fine.